code/stack/stack.py

```
@@ -176,7 +180,79 @@ def reverse_list(data: list) -> list:
+    DEFAULT_CAPACITY = 10
+
+    def __init__(self):
+
+        self._data = [None] * ArrayQueue.DEFAULT_CAPACITY
+        self._size = 0
+        self._front = 0
+
+    def first(self) -> int:
+
+        if self.isEmpty():
+            raise Empty("Queue is empty")
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)